src/app/modules/main/chat_section/chat_content/module.nim

```diff
@@ -205,6 +205,13 @@ method getMyChatId*(self: Module): string =
 method isMyContact*(self: Module, contactId: string): bool =
   self.controller.getMyAddedContacts().filter(x => x.id == contactId).len > 0
 
+method isMember*(self: Module, contactId: string): bool =
```

Comment:
@borismelnik checking if contact with a certain id is member of this chat or not, from my point of view is totally fine  to check that in a way you did here, since `chatService` (`chats` table maintained there) is our source of truth in terms of everything chat related. All our models including users list are fulfilled from data coming from different services.